packages/core/src/components/overlay/overlay.tsx

```diff
@@ -193,7 +193,14 @@ export class Overlay extends React.Component<IOverlayProps, IOverlayState> {
         };
 
         if (inline) {
-            return <span {...elementProps} ref={this.refHandlers.container}>{transitionGroup}</span>;
+            return (
+                <span
+                    {...elementProps}
+                    ref={this.refHandlers.container}
+                >
+                    {transitionGroup}
+                </span>
```

Comment:
Much easier to parse next to the corresponding `else` block right after, don't you think?